cmd_historgram.sh

```diff
@@ -28,66 +28,66 @@ do
     exit 0
     ;;
   n)
-    NUM_ENTRIES=$OPTARG
+    num_entries=$OPTARG
     ;;
   f)
-    HISTFILE=$OPTARG
+    histfile=$OPTARG
     ;;
   c)
-    CHART_CHAR="$OPTARG"
+    chart_char="$OPTARG"
     ;;
   l)
-    LINE_LEN=$OPTARG
+    line_len=$OPTARG
     ;;
   esac
 done
 
-shift $((OPTIND-1))
+shift $((optind-1))
 
 [ "${1:-}" = "--" ] && shift
 
-echo "entries=$NUM_ENTRIES, file=$HISTFILE, char=$CHART_CHAR, len=$LINE_LEN"
+echo "entries=$num_entries, file=$histfile, char=$chart_char, len=$line_len"
 
-CALC=$(cat $HISTFILE | awk '{print $1}' | sort | uniq -c | sort -rn)
+calc=$(cat $histfile | awk '{print $1}' | sort | uniq -c | sort -rn)
 
-for (( N=0; N<=$NUM_ENTRIES; N++ ))
+for (( n=0; n<=$num_entries; n++ ))
 # gather counts and cmds
 do
-  CMDS[$N]=$(echo "$CALC" | sed -ne "`expr $N + 1`p")
-  COUNTS[$N]=$(echo ${CMDS[$N]} | awk '{print $1}')
-  S=$(echo ${CMDS[$N]} | cut -d' ' -f2-)
-  MAX_LEN=$((
-  ${#S} > $MAX_LEN ?
-    ${#S}:
-    $MAX_LEN
+  cmds[$n]=$(echo "$calc" | head -n `expr 1 + $n` | tail -n 1)
```

Comment:
can you revert this line to the `sed` command from the previous commit? 

thanks
```suggestion
  cmds[$n]=$(echo "$calc" | sed -ne "`expr 1 + $n`p")
```